MeesGame/Gameplay/GameObjects/TilesTypes/PortalTile.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Microsoft.Xna.Framework;
+using System.ComponentModel;
+
+namespace MeesGame
+{
+    class PortalTile : FloorTile
+    {
+        protected int portalIndex;
+        protected Point destination;
+
+        public PortalTile(int layer = 0, string id = "") : base(TileType.Portal, layer, id)
+        {
+        }
+
+        [Editor]
+        public int PortalIndex
+        {
+            get
+            {
+                return portalIndex;
+            }
+            set
+            {
+                portalIndex = value;
+                TileData tileData = Data;
+                tileData.AdditionalInfo = portalIndex;
+                Data = tileData;
+            }
+        }
+
+        public override bool IsActionForbiddenFromHere(ITileFieldPlayer player, PlayerAction action)
+        {
+            // All actions are allowed on a PortalTile.
+            return false;
```

Comment:
This should return false if action==special, and call base.isactionforbiddenfromhere otherwise.